Update Miner Docs for Latest Node Versions (#1616)
stacks-network/docs: stacks-in-depth/nodes-and-miners/mine-mainnet-stacks-tokens.md

```json
{
  "mnemonic": "stomach strong dog warfare drip garbage vendor among argue flash anchor change any credit purity learn gas unfold urban electric van alcohol lobster reflect",
  "keyInfo": {
    "privateKey": "4b48f79eea88bb20de75148ecc8e5480463048b937d9c77801b388b4e1b49d2d01",
    "publicKey": "03a707458e468f7ee6513f8135500780576dca2b52c98ae909a538f75e4901aad5",
    "address": "SP2D9W82GHM1V0ACDRVCDNKJV0J6NS7KJ25WHHRMH",
    "btcAddress": "1F6YVM1E633QDUo9zgT3q7K3T9FYuKJ3RZ",

>     "wif": "Kyk49jsPGen5C1ThhyJJH4CndLk8yLESuQJVGsbbTV3FFF9CRTJG",

Reviewer: I would prefer to use the same fake values in the miner-docs repo here. `1F6YVM1E633QDUo9zgT3q7K3T9FYuKJ3RZ` is a valid address

Author: The old `make_keychain` didn't have a `wif` field, so i updated them with the new ones. Should i change the generated keychain back to the old one and just add a random wif?

Reviewer: intersting. it should have since it's part of the output. 
i would use the psuedo values from the miner-docs repo here for both mainnet and testnet sections

Author: Got it, will do that, thank you!

    "index": 0
  }
}
```

**Warning:** **Do not lose this information** - we'll need to use the `privateKey`, `btcAddress` and `wif` fields in later steps.

The above `wif` (`Kyk49jsPGen5C1ThhyJJH4CndLk8yLESuQJVGsbbTV3FFF9CRTJG`) will then need to be imported into the bitcoin mainnet network.

Next, a bitcoin wallet is created:
```bash
bitcoin-cli \
  -rpcconnect=127.0.0.1 \
  -rpcport=8332 \
  -rpcuser=btcuser \
  -rpcpassword=btcpass \
  createwallet "miner" \
  false \
  false \
  "" \
  false \
  false \
  true
```

Now, import your wif (bitcoin private key) inside the newly created wallet.

> **Note:** Be sure to replace `<wif from JSON above>` with the bitcoin address in the `Generate a keychain` step.

Reviewer: ```suggestion
**Note:** Be sure to replace `<wif from JSON above>` with the wif value in the `Generate a keychain` step.
```

working_dir = "/stacks-blockchain"
rpc_bind = "0.0.0.0:20443"
p2p_bind = "0.0.0.0:20444"
seed = "<privateKey from JSON above>>"

> local_peer_seed = "<privateKey from JSON above>>"

Reviewer: ```suggestion
```

Alternatively, you can run a Stacks mainnet miner with Docker.

**Caution:** Ensure you have [Docker](https://docs.docker.com/get-docker/) installed.

#### Generate a Keychain and Get Some Tokens

Generate a keychain:

```bash
docker run -i node:20-alpine npx @stacks/cli make_keychain 2>/dev/null | jq -r
```

We need to get some BTC to that address. You should be able to transfer BTC to this address using a cryptocurrency exchange such as [Coinbase](https://www.coinbase.com), [Binance](https://www.binance.com), or [Kraken](https://www.kraken.com).

#### Update Stacks Blockchain Docker Configuration File

Use the steps oulined above to create the configuration file.

#### Start the Stacks Blockchain miner with Docker

> **Info:** The ENV VARS `RUST_BACKTRACE` and `STACKS_LOG_DEBUG` are optional. If removed, debug logs will be disabled.

Reviewer: @kenrogers

  -p 20443:20443 \
  -p 20444:20444 \
  blockstack/stacks-blockchain:latest \

>   /bin/stacks-node start --config /src/stacks-node/mainnet-miner-conf.toml

Reviewer: ```suggestion
/bin/stacks-node start --config /src/stacks-node/mainnet-miner-conf.toml
```
